packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -335,27 +335,20 @@ contract ValidatorManager is ISemver, IValidatorManager {
     }
 
     /**
-     * @notice Returns the commission rate of given validator.
-     *
-     * @param validator Address of the validator.
-     *
-     * @return The commission rate of given validator.
+     * @inheritdoc IValidatorManager
      */
     function getCommissionRate(address validator) external view returns (uint8) {
         return _validatorInfo[validator].commissionRate;
     }
 
     /**
-     * @notice Returns the pending commission rate of given validator.
-     *
-     * @param validator Address of the validator.
-     *
-     * @return The pending commission rate of given validator.
+     * @inheritdoc IValidatorManager
      */
     function getPendingCommissionRate(address validator) external view returns (uint8) {
         return _validatorInfo[validator].pendingCommissionRate;
     }
 
+    // TODO: move this to AssetManager
```

Comment:
Is the reason why just marking TODO is that this PR is just related to interface?

Reply:
I applied the changes.